src/System.IO.Abstractions.TestingHelpers/MockFileData.cs

```diff
@@ -10,6 +10,8 @@ namespace System.IO.Abstractions.TestingHelpers
     [Serializable]
     public class MockFileData
     {
+        public const long LengthIsNotFaked = -1;
```

Comment:
Codacy found an issue: [Change this constant to a 'static' read-only property.](https://app.codacy.com/gh/System-IO-Abstractions/System.IO.Abstractions/pullRequest?prid=6242633)

Reply:
OK. Fixed.